include/aca_oam_server.h

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_OAM_SERVER_H
+#define ACA_OAM_SERVER_H
+
+#include <cstdint>
+#include <string>
+#include <arpa/inet.h>
+#include <net/ethernet.h>
+#include <netinet/ether.h>
+
+#include "goalstateprovisioner.grpc.pb.h"
+
+using namespace std;
+namespace aca_oam_server
+{
+//OAM Message Type
+//#define OAM_MSG_NONE
+#define SIZE_OP_CODE 8
+#define OAM_SOCKET_PORT 8300
```

Comment:
note that the OAM port is configured by Zeta, therefore you will need to remove this static define and use the information from the AuxGateway message:

    message zeta {
        // port for in-band (same NIC channel) operation
        uint32 port_inband_operation = 1;
    }

Reply:
+1. Let us try hard first not to hard code at all. If really need it for some reason, let us keep the constant variable at a centralized place e.g. in a constant file so that we can easily track and remove them in the future.

Reply:
yes, we have /include/aca_config.h to store any constant if needed.